docs/index.html

```diff
@@ -10,25 +7,19 @@
     <!-- Bootstrap CSS -->
     <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.2/css/bootstrap.min.css" integrity="sha384-Smlep5jCw/wG7hdkwQ/Z5nLIefveQRIY9nfy6xoR1uRYBtpZgI6339F5dgvm/e9B" crossorigin="anonymous">
     <style>
-        body,
-        html {
```

Comment:
omitting this creates this effect in apples's browsers: 
<img width="716" alt="Screen Shot 2020-01-16 at 1 19 40 PM" src="https://user-images.githubusercontent.com/3653760/72563581-d59ce080-3862-11ea-8f93-8b94b8a3ef69.png">

Again, limit commits to one change.